### PR TITLE
Zowe chat docs

### PR DESCRIPTION
**Pull request by nannanli**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/troubleshoot/zowe-chat-troubleshoot/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Troubleshooting Zowe Chat
+
+As a Zowe Chat user, you might encounter some issues with how Zowe Chat works. This section lists some solutions to help you. 
+
+## Check the chatServer.log
+
+When you run into some errors, you can check the server log which can be found in `$ZOWE_CHAT_HOME/chatServer.log/`.
```
Comment (XuehongBJLiang) [inline docs/troubleshoot/zowe-chat-troubleshoot/troubleshooting.md:7]: Hi. log directory is $ZOWE_CHAT_HOME/log and log file is $ZOWE_CHAT_HOME/log/chatServer.log

File: docs/user-guide/zowe-chat/systemrequirements-chat.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    -   Linux on System x
+    -   Linux on System z
+
+- [Node.js]((https://nodejs.org/en/)) v16.13.2 or later is installed on your Linux server
+
+- Chat platform: 
+
+   - Mattermost 7.0 or later
+
+     See [Configuring Mattermost chat platform](chat_prerequisite_mattermost.md).
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/systemrequirements-chat.md:41]: The links have some issues with urlencoder

File: docs/user-guide/zowe-chat/chat_configure_teams.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   ```
+   #  Specify the bot user name or account that is used in your chat tool server.
+   # The default value is zowechat.
+   botUserName: zowechat
+
+   # Specify the ID of your Microsoft Teams bot.
+   botId: <Your Bot ID>
+
+   # Specify the password of your Microsoft Teams bot.
+   # The password will be encrypted automatically by Z ChatOps server with <AES> prefix added.
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/chat_configure_teams.md:34]: delete

File: docs/user-guide/zowe-chat/chat_configure_mattermost.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+   # Specify the team URL of your team created in your Mattermost server.
+   # The default value is devops.
+   teamUrl: devops
+
+   # Specify the bot user name or account that is used in your chat tool server.
+   # The default value is zowechat.
+   botUserName: zowechat
+
+   # Specify the access token to connect to your bot.
+   # The token will be encrypted automatically by Z ChatOps server with <AES> prefix added.
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/chat_configure_mattermost.md:75]: delete

File: docs/user-guide/zowe-chat/systemrequirements-chat.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- Disk space: 300 M for native installation
+- Port numbers: 2
+  - 4001 for Microservice
+  - 5001 for for messaging endpoint of Chatbot
+
+   :::note
+   
+   You can configure the port number in the configuration file as you need. 
+   
+   To configure the port number for Microservice, edit the Microservice server configuration file<!--at ZCHATOPS_HOME/microservice/config/bnzsvc-server.yaml-->.
```
Comment (samanthasusu) [inline docs/user-guide/zowe-chat/systemrequirements-chat.md:20]: microservice -> web app
Path: “zchatops”

**Review by samanthasusu — CHANGES_REQUESTED** [review]

**Review by samanthasusu — APPROVED** [review]
LGTM! Thanks

**Pull request merged**